entries/p/portal.arubainstanton.com.json

```
@@ -0,0 +1,12 @@
+{
+  "ArubaInstantOn": {
```

```suggestion
  "Aruba Instant On": {
```

The name appears as "Aruba Instant On" for me